SConstruct

```diff
@@ -1536,43 +1546,44 @@ if env['python_package'] != 'none':
-                    "{}, but {} or newer is required.".format(
-                        ruamel_yaml_version, ruamel_min_version))
+                logger.error(
+                    "ruamel.yaml is an incompatible version: Found "
+                    f"{ruamel_yaml_version}, but {ruamel_min_version} or newer "
+                    "is required.")
                 sys.exit(1)
 
     if warn_no_python:
-        if env['python_package'] == 'default':
-            print('WARNING: Not building the Python package because the Python '
-                  'interpreter {!r} could not be found'.format(env['python_cmd']))
-            env['python_package'] = 'none'
+        if env["python_package"] == "default":
+            logger.warning(
+                "Not building the Python package because the Python interpreter "
+                f"'{env['python_cmd']!r}' could not be found.")
+            env["python_package"] = "none"
         else:
-            print('ERROR: Could not execute the Python interpreter {!r}'.format(
-                env['python_cmd']))
+            logger.error(
+                f"Could not execute the Python interpreter '{env['python_cmd']!r}'")
             sys.exit(1)
     elif python_version < python_min_version:
-        msg = ("{}: Python version is incompatible. Found {} but {} "
-               "or newer is required")
-        if env["python_package"] in ("minimal", "full"):
-            print(msg.format("ERROR", python_version, python_min_version))
+        msg = ""
+        if env["python_package"] in ("minimal", "full", "default"):
+            logger.error(
+                f"Python version is incompatible. Found {python_version} but "
+                f"{python_min_version} or newer is required. In order to install "
+                "Cantera without Python support, specify 'python_package=none'.")
             sys.exit(1)
-        elif env["python_package"] == "default":
-            print(msg.format("WARNING", python_version, python_min_version))
-            env["python_package"] = "none"
-    elif env['python_package'] == 'minimal':
-        # If the minimal package was specified, no further checking
-        # needs to be done
-        print('INFO: Building the minimal Python package for Python {}'.format(python_version))
+    elif env["python_package"] == "minimal":
+        # If the minimal package was specified, no further checking needs to be done
+        logger.info(f"Building the minimal Python package for Python {python_version}")
     else:
 
         if len(info) > expected_output_lines:
-            print("WARNING: Unexpected output while checking Python "
-                  "dependency versions:")
-            print('| ' + '\n| '.join(info[expected_output_lines:]))
+            msg = "\n| ".join(info[expected_output_lines:])
+            logger.warning(
+                f"Unexpected output while checking Python dependency versions:{msg}")
```

Comment:
I think this is now going to print the first line of the unexpected output immediately after "...dependency versions", rather than on the following line with the `| ` prefix that is done for the following lines.

Reply:
:+1: Oops.